packages/transport-webrtc/package.json

```
@@ -68,17 +70,16 @@
     "uint8arrays": "^4.0.4"
   },
   "devDependencies": {
-    "@chainsafe/libp2p-yamux": "^4.0.1",
-    "@libp2p/interface-compliance-tests": "^3.0.0",
-    "@libp2p/peer-id-factory": "^2.0.0",
-    "@libp2p/websockets": "^6.0.0",
+    "@chainsafe/libp2p-yamux": "^4.0.2",
+    "@libp2p/peer-id-factory": "^2.0.4",
+    "@libp2p/websockets": "^6.0.3",
     "@types/sinon": "^10.0.15",
     "aegir": "^39.0.10",
     "delay": "^6.0.0",
     "it-length": "^3.0.2",
     "it-map": "^3.0.3",
     "it-pair": "^2.0.6",
-    "libp2p": "^0.45.0",
+    "libp2p": "^0.45.9",
```

Not necessary as this is a monorepo sibling and this change will be undone by `aegir check-project`.

```suggestion
    "libp2p": "^0.45.0",
```